src/d3a/models/state.py

```diff
@@ -234,12 +234,3 @@ def market_cycle(self, past_time_slot, time_slot):
         self.charge_history[time_slot] = 100.0 * self.used_storage / self.capacity
         self.charge_history_kWh[time_slot] = self.used_storage
         self.offered_history[time_slot] = self.offered_sell_kWh[time_slot]
-
```

Comment:
Can you tell me the reason for reverting this again?
I can remember that I was asking for the reason for this separate class in one of you previous PRs and @spyrostz answered:
https://github.com/gridsingularity/d3a/pull/361#discussion_r249437805

Did something change in the meantime?

Reply:
https://github.com/gridsingularity/d3a/pull/361/files#r249868813

Reply:
Yes, but the mentioned comment states only that the MarketClearingState class should be moved to a different file, not deleted totally. Right?

Reply:
Yes, I have moved it to market_structures in my last commit https://github.com/gridsingularity/d3a/pull/366/files#diff-286711868906f1b84a065750ab4e674fR164

Reply:
Ok! Thanks! No i see it. An explanatory sentence is always better than a link ;-)